server/index.js

```diff
@@ -94,6 +94,12 @@ app.init = function({
         filter: contentType => !contentType.startsWith('text/event-stream'),
       })
     )
+    // this doesn't work
+    //.use(async function(ctx) {
+    //   ctx.authTokenHeaders = {
+    //     'token-name': ctx.headers['token-injected'],
+    //   };
+    // })
```

Comment:
Middleware takes a function with arguments `ctx`, `next`
You will need to execute the next callback for this to work.
For example:
```await next();```